.github/ISSUE_TEMPLATE/blog_post_ga_release.md

```diff
@@ -35,4 +35,4 @@ If you have previously provided this information for an Open Liberty beta blog p
 
 - Your paragraph will be included in the GA release blog post. It might be edited for style and consistency.
 - You will be asked to review a draft before publication.
-- If you would _also_ like to write a standalone blog post about your update (highly recommended), raise an issue on the [Open Liberty blogs repo](https://github.com/OpenLiberty/blogs/issues/new/choose). State in the issue that the blog post relates to a specific release so that we can ensure it is published on an appropriate date (it won't be the same day as the GA blog post).
\ No newline at end of file
+- If you would _also_ like to write a standalone blog post about your update (highly recommended), raise an issue on the [Open Liberty blogs repo](https://github.com/OpenLiberty/blogs/issues/new/choose). State in the issue that the blog post relates to a specific release so that we can ensure it is published on an appropriate date (it won't be the same day as the GA blog post).
```

Comment:
My spot the difference skills are failing me! These look the same?

Reply:
I agree. No idea why it's showing that para as changed. It was there already.